### PR TITLE
114-m1.5: new warning alert added when a specific key was set but not…

### PR DESCRIPTION
**Pull request by Rspillere**
new warning alert added when a specific key was set but not sended

### CODE REVIEW

File: packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js [hunk 1]
```diff
@@ -14,6 +16,16 @@ const keyValValidator = (cmcdJson, errors, warnings, config, warningFlag = true)
     checkSidIsPresent(cmcdJson, warnings);
   }
 
+  // Check if all the specifickey have been received in the request
+  if (config?.specificKey) {
+    const keysReceved = Object.keys(cmcdJson);
+    config.specificKey.forEach((key) => {
+      if (!keysReceved.includes(key)) {
+        warnings.push(createWarning(warningTypes.specificKeysNotSended, key));
```
Comment (rodrigo-mira) [inline packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js:24]: Typo here, it should be 'sent'

Comment (Rspillere) [reply packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js:24]: thank you! I'm going to change it right now

**Review by JoaquinBCh — CHANGES_REQUESTED** [review]
Could you change double quotes to single in queryConfigTestCases?

File: packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js [hunk 2]
```diff
@@ -14,11 +16,30 @@ const keyValValidator = (cmcdJson, errors, requestID, warnings, config, extended
     checkSidIsPresent(cmcdJson, warnings, requestID);
   }
 
+  // Check if all the specifickey have been received in the request
+  const keysReceved = Object.keys(cmcdJson);
```
Comment (mcamardon) [inline packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js:20]: keysReceived

**Review by JoaquinBCh — APPROVED** [review]

**Pull request merged**